ext/AvizExt/viz/sensitivity.jl

```diff
@@ -2,6 +2,10 @@ using Statistics
 using Printf
 using ADRIA.sensitivity: _get_cat_quantile
 
+const FV_LABELS = [
+    "cf", "unguided", last.(split.(string.(ADRIA.decision.mcda_methods()), "."))...
+]
```

Comment:
Suggest changing this to a function that returns the string vector to avoid having to restart Julia if the constant needs to be changed.